docs/magento2/search-engine.mdx

```diff
@@ -228,6 +229,102 @@ query Search {
 If you are using the default theme or the Chocolatine theme, the search bar
 should now be visible.
 
+## Attraqt
+
+> _This feature has been added in Front-Commerce `2.19.0`_
```

Comment:
Why not a simple message like in other pages ?  (e.g. maintenance mode, password fields, etc)

```suggestion
> _Since version 2.19_
```